se/se_epub_generate_toc.py

```diff
@@ -128,40 +130,20 @@ def landmark_link(self, work_type: str = "fiction", work_title: str = "WORK_TITL
 
 		return out_string
 
-def get_file_epub_type(soup: BeautifulSoup) -> str:
-	"""
-	Retrieve the epub_type of this file to see if it's a landmark item.
-
-	INPUTS:
-	soup: BeautifulSoup representation of the file
 
-	OUTPUTS:
-	the epub_type, eg: "dedication", "epigraph", etc.
+def get_first(node: EasyXmlTree, wanted: str):
```

Comment:
Is this function now unused? Can we remove it?

Reply:
Yes, will do.